staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch_test.go

```diff
@@ -1838,6 +1845,28 @@ testCases:
     result:
       name: 1
       value: 1
+  - description: defined null values should propagate removing original fields
```

Comment:
@pwittrock added this test-case for the non-conflicting case applying nulls to original fields. PTAL